staff/kapil-siva/course-1/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>NYSL</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="second" href="index1.html" />
+  </head>
+  <body>
+    <img src="nysl_logo.png" alt="NYSL logo" />
+    <h1 class="heading">Northside Youth Soccer League</h1>
+    <ul class="navigation">
+      <li><a href=index.html>Home</a></li>
+      <li><a href=index1.html>About NYSL</a></li>
+      <li><a href=index2.html>Contact</a></li>
+    </ul><br>
+    <h1 class="event">Upcoming Events</h1>
+    <img class="img1" src="soccer_image1.jpg" />
+    <ul class="upcoming">
```

Comment:
use a more accurate class name like for instance `upcoming-events` instead of just `upcoming`